atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformerJS.kt

```diff
@@ -16,7 +16,7 @@ private const val ATOMIC_ARRAY_CONSTRUCTOR = """Atomic(Ref|Int|Long|Boolean)Arra
 private const val MANGLED_VALUE_PROP = "kotlinx\$atomicfu\$value"
 
 private const val TRACE_CONSTRUCTOR = "atomicfu\\\$Trace\\\$"
-private const val TRACE_APPEND = "atomicfu\\\$Trace\\\$append\\\$"
+private const val TRACE_APPEND = """atomicfu\$(Trace)\$(append)\$(1|2|3|4)\$"""
```

Comment:
Please ensure that new methods are correctly eliminated from JS

Reply:
I've added the test to check hat no references to atomicfu are left in the js output